frontend/components/top_nav/SiteTopNav/SiteTopNav.tsx

```diff
@@ -54,11 +54,6 @@ const REGEX_GLOBAL_PAGES = {
   PROFILE: /\/profile/i,
 };
 
-const REGEX_EXCLUDE_NO_TEAM_PAGES = {
-  MANAGE_POLICIES: /\/policies\/manage/i,
-  MANAGE_QUERIES: /\/queries\/manage/i,
```

Comment:
Was a little worried about removing this, but the link just includes `team_id=0` and when it realizes we don't support queries team_id=0, it kicks the user to queries without a team_id with no extra API calls so we're good <3